README.md

```diff
@@ -113,7 +113,7 @@ We'd love you to contribute to Linuxbrew or its upstream, Homebrew. First, pleas
 
```

Comment:
Just noticed that we should probably update the "Something broke!" section to reflect that there are now 2 repos (core vs brew)